### PR TITLE
A couple of spelling fixes.

### PR DESCRIPTION
**Pull request by revehiet**

### CODE REVIEW

File: ui.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def draw(self, context):
         else:
             uv_obj = get_object(context, context.object)
 
-        labeltext = "baking UV from " + uv_obj.name
+        labeltext = "Bake UVs of " + uv_obj.name
```
Comment (u3dreal) [inline ui.py:171]: should be from since it indicates the second selected object when baking from to

File: ui.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def draw(self, context):
             row = layout.row()
             row.label(text="MolObject: " + obj.name)
             row = layout.row()
-            row.prop(psys, "mol_voxel_size", text="Voxel Size")
+            row.prop(psys, "mol_voxel_size", text="Resolution")
```
Comment (u3dreal) [inline ui.py:104]: Voxel size is better not to confuse with particle system grid resolution

**Comment by u3dreal** [comment]
plz merge with main

File: addon_prefrences.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ class MolecularAddonPreferences(AddonPreferences):
     )
 
     use_retina: BoolProperty(
-        name='retina display',
+        name='Use correct particle size for Apple devices.',
```
[the remark this answers is not in the document]
Comment (revehiet) [reply addon_prefrences.py:18]: "retina display" is not descriptive enough. Maybe "Use correct particle size for Retina displays." is better?

File: ui.py (hunk 2 above)
Comment (revehiet) [reply ui.py:104]: But particle system grid resolution _is_ what we are changing? hence why "Resolution" is better.

File: ui.py (hunk 1 above)
Comment (revehiet) [reply ui.py:171]: 👌

File: addon_prefrences.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (HeCorr) [reply addon_prefrences.py:18]: could leave `Retina Display` as the name and put the more descriptive text as the description (which shows up when hovering over it)

**Comment by u3dreal** [comment]
Anythign going on here ? If not i will close soon

**Pull request closed without merging**